### ISSUE TITLE
Space should be trigger menu items (open them, check / uncheck them...)

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#92895]
Issue Type: <b>Bug</b>

Steps to reproduce:
1. Open settings.json and include the following line:
> "window.titleBarStyle": "custom"
2. Save the configuration and restart VSCode.
3. Try to check/uncheck auto save in file menu.

At least in my environment nothing happens.
@isidorn  Could this have anything to do with accessibility?

VS Code version: Code - Insiders 1.44.0-insider (b7bfce30be3c30aa2c33fbe94fe82b81d4105668, 2020-03-17T05:37:36.728Z)
OS version: Linux x64 5.5.8-arch1-1

**Comment by isidorn** [comment]
@jvesouza this is due to custom menus on Linux.
I can check / uncheck it using the `enter` key (and that closes the menu). However `space` key does not check / uncheck.
Which seems to be the actual issue, thus renaming this and assigning to @sbatten since he owns the custom menus.

**Comment by jvesouza** [comment]
@isidorn  There must definitely be something wrong in my environment because I can't even check with the space bar or the enter key. I'm using today's version of code-insiders.

@ajborka  @joanmarie When time permits, would it be possible to test to confirm that the problem is in my environment? Thank you in advance.

**Comment by jvesouza** [comment]
Hi all.
I found the cause of the problem, and as usually happens in these cases, the problem was between the chair and the keyboard.
When changing the settings.json file I ended up not putting a comma at the end of the line it includes.
Instead of: 
"window.titleBarStyle": "custom",
I used:
"window.titleBarStyle": "custom"


Please forgive me for the noise!

**Comment by isidorn** [comment]
No problem, thanks for letting us know. That is a common error that I often hit.
However my initial comment still holds: space should trigger menu items in the custom menu. Let's see what @sbatten says.

**Comment by ajborka** [comment]
@isidorn  I agree with you on the menu problem. Most other apps allow spacebar to activate menu items. Trying it in Mate or Gnome should demonstrate my point.

**Comment by jvesouza** [comment]
@isidorn  Thank you and I agree with your initial comment.
On the other hand I was wondering if VSCode should not have given an error message when saving the settings.json file, since there was an error in it.

**Comment by isidorn** [comment]
@jvesouza for the settings.json issue missing error can you please file a new issue with repro steps and ping me on it - if you can reproduce it reliably and then I can forwad it to my collegue Martin who is in charge of json validation.

**Comment by sbatten** [comment]
To Verify:
1. Open VS Code on Linux.
2. Enable the custom titlebar with `window.titleBarStyle`
3. Use keyboard to navigate the file menu.
4. Verify that space activates `Auto Save`

**Comment by jvesouza** [comment]
@sbatten @isidorn  I confirm that it has been corrected, great!
Thanks.

**Comment by isidorn** [comment]
@jvesouza thanks for trying it out
@sbatten thanks a lot for fixing it!

Adding verified label.
